staging/src/k8s.io/legacy-cloud-providers/azure/cache/doc.go

```diff
@@ -1,6 +1,3 @@
-//go:build !providerless
-// +build !providerless
```

Comment:
Why is this "unnecessary"?
Tagging the legacy providers like this ensures we can't compile code that imports them when the tag is set ...?

Reply:
@BenTheElder these are `doc.go` files. they don't have any code in them. some of the scripts need something in the package or they will complain `build constraints exclude all Go files`..